core/controllers/collection_editor.py

```diff
@@ -91,7 +91,7 @@ def put(self, collection_id):
 
         commit_message = self.payload.get('commit_message')
 
-        if len(commit_message) > 1000:
+        if len(commit_message) > feconf.MAX_LENGTH_OF_COMMIT_MESSAGES:
```

Comment:
MAX_COMMIT_MESSAGE_LENGTH (otherwise, sounds like the max length of a list of messages)

Reply:
Done